bridges/snowbridge/pallets/ethereum-client/src/lib.rs

```diff
@@ -464,13 +469,24 @@ pub mod pallet {
 				Self::deposit_event(Event::SyncCommitteeUpdated {
 					period: update_finalized_period,
 				});
+				sync_committee_updated = true;
```

Comment:
This `sync_committee_updated` flag seems redundant. Following the code, when `sync_committee_updated=True`, it follows that `update.next_sync_committee_update == Some(...)`